packages/blade/src/components/Checkbox/CheckboxIcon/CheckboxIcon.tsx

```diff
@@ -8,27 +8,71 @@ import { useTheme } from '~components/BladeProvider';
 import Svg, { Path } from '~components/Icons/_Svg';
 import { getIn } from '~utils';
 
-const CheckedIcon = ({ color }: { color: string }) => {
+const svgSize = {
```

Comment:
**nit (but also lets do this 🙈 ):** Imo it would be better and more readable if we have `CheckedIconSmall`, `CheckedIconMedium`, etc. Having this logic for width, height, viewBox, d makes it seem too complicated. Lets treat it as an icon for small checkbox and an icon for medium checkbox?

Reply:
Re: Lets do this instead

https://github.com/razorpay/blade/pull/758#discussion_r1004392364

Reply:
We were discussing this. 

https://github.com/razorpay/blade/pull/758#discussion_r1004031973

We did the same way earlier.